pages/events/index.vue

```diff
@@ -97,7 +97,10 @@
         </template>
         <template slot="extra-info">
           <div class="event-page__section">
-            <h3>Follow our event calendar</h3>
+            <a href="#follow-our-event-calendar" />
```

Comment:
What does this empty link achieve?

Reply:
This is how I usually add the different anchors to the URL, by adding a link tag that is hidden. I'm open to suggestions on how to do it less messy.

Reply:
I had the same question as @eddybrando 

I don't think it's needed. Try removing it and still visiting http://localhost:3000/events#follow-our-event-calendar

the id in this element `<div id="follow-our-event-calendar" class="event-page__section">` _should_ allow users to get to that section

unless I'm missing some effect that we're going for

Reply:
You're both right! just removed it